django_dramatiq/management/commands/rundramatiq.py

```
@@ -152,31 +154,47 @@ def is_ignored_module(module_name):
             return False
 
         for conf in app_configs:
-            module = conf.name + ".tasks"
-
-            if is_ignored_module(module):
-                self.stdout.write(" * Ignored tasks module: %r" % module)
-                continue
-
-            imported_module = importlib.import_module(module)
-            if not self._is_package(imported_module):
-                self.stdout.write(" * Discovered tasks module: %r" % module)
-                tasks_modules.append(module)
-            else:
-                submodules = self._get_submodules(imported_module)
-
-                for submodule in submodules:
-                    if is_ignored_module(submodule):
-                        self.stdout.write(" * Ignored tasks module: %r" % submodule)
-                    else:
-                        self.stdout.write(" * Discovered tasks module: %r" % submodule)
-                        tasks_modules.append(submodule)
+            for task_module in DRAMATIQ_AUTODISCOVER_MODULES:
+                if conf.name == "django_dramatiq":
+                    # we want to find our own tasks, even if the user has overridden
+                    # the tasks module name
+                    module = "django_dramatiq.tasks"
+                else:
+                    module = conf.name + "." + task_module
+
+                if is_ignored_module(module):
+                    self.stdout.write(" * Ignored tasks module: %r" % module)
+                    continue
+
+                try:
+                    imported_module = importlib.import_module(module)
```

I would prefer to let this throw an exception, as before.